server/engine/Game.js

```diff
@@ -0,0 +1,124 @@
+const wordArray = require('./Words');
+
+class Game {
+  ADD_POINT = 100;
+
+  /**
+   * @param {string} id : Room Name 
+   * @param {array} players : game players array
+   */
+  constructor(id, players) {
+    this.roomName = id;
+    this.guesser = '';
+    this.word = '';
+    this.point = 0;
+    this.round = 0;
+    this.wordArray = [];
+    this.players = players;
```

Comment:
Maybe it is better to have separate Player class with **id** and **isGuesser** properties

Reply:
I agree with @InsafKhamzin. Should make it easier to determine how points are distributed to each player.

Reply:
@InsafKhamzin @darrenMabbayad I added Player.js and changed that point will be calculated individually.